backstory/src/main/java/com/google/sps/servlets/PerspectiveServlet.java

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import au.com.origma.perspectiveapi.v1alpha1.PerspectiveAPI;
+import com.google.gson.Gson;
+import com.google.sps.data.PerspectiveAnalysis;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that filters text using the Perspective API. */
+@WebServlet("/perspective")
+public final class PerspectiveServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I'd recommend using POST instead of GET here. In my experience GET by convention tends to be for retrieving some existing resource. POST is more flexible, and would be more conventionally be used for tasks like data processing (like this use case).

In addition, GET has more limitations on the size of the message input (e.g. if in the future you support longer stories, GET may no longer be compatible).

cc: @aaronegolden in case he has other opinions about this

Reply:
I will change this to a post method. I wasn't sure if it would be more conventionally a GET or POST method, as it does retrieve the data for display as well as processing it. However, now that I now, I will change it to a POST method. This servlet won't be used in the future, but I still think it's worthwhile to make the change.

Reply:
Done